include/sys/fs/zfs.h

```diff
@@ -115,7 +115,8 @@ typedef enum {
 	ZFS_PROP_READONLY,
 	ZFS_PROP_ZONED,
 	ZFS_PROP_SNAPDIR,
-	ZFS_PROP_PRIVATE,		/* not exposed to user, temporary */
+	ZFS_PROP_PRIVATE,		/* not exposed to the user, temporary */
+	ZFS_PROP_ACLMODE,
```

Comment:
While this was the original placement, we should add this to the end to avoid reordering the enum.  There may be external consumer depending on it.